src/codecatalyst/activation.ts

```diff
@@ -96,3 +98,29 @@ export async function activate(ctx: ExtContext): Promise<void> {
         authProvider.removeSavedConnection()
     })
 }
+
+async function showReadmeFileOnFirstLoad(workspaceState: vscode.ExtensionContext['workspaceState']): Promise<void> {
+    // Check dev env state to see if this is the first time the user has connected to a dev env
+    const isFirstLoad = workspaceState.get('aws.codecatalyst.devEnv.isFirstLoad', true)
+
+    if (!isFirstLoad) {
+        return
+    }
+
+    const readmeUri = vscode.workspace.findFiles('README.md').then(files => {
+        if (files.length === 0) {
+            return undefined
+        }
+        return files[0]
+    })
+
+    if (readmeUri === undefined) {
```

Comment:
does `findFiles..then` need to be await'd ?

Reply:
My bad, made a change right before pushing. I'm setting up to test in C9 right now, so will catch if anything else breaks.

Fixing this